src/scrabble-long.js

```diff
@@ -0,0 +1,134 @@
+      if (this.word[i] === '{') {
+        doublePoints = true
+        i++
+      }
+      // if the character is "}" or "]" then double points will swithc to FALSE therefore the letter after this character only worth single points.
+      if (this.word[i] === '}') {
+        doublePoints = false
+      }
+
+      if (this.word[i] === '[') {
+        triplePoints = true
+        i++
+      }
+
+      if (this.word[i] === ']') {
+        triplePoints = false
+      }
+
+      if (
+        pointOne.includes(this.word[i]) ||
+        pointOne.includes(this.word[i].toLowerCase())
+      ) {
+        if (doublePoints === true) {
+          this.points += 1 * 2
+        } else if (triplePoints === true) {
+          this.points += 1 * 3
+        } else {
+          this.points += 1
+        }
+      } else if (
+        pointTwo.includes(this.word[i]) ||
+        pointTwo.includes(this.word[i].toLowerCase())
+      ) {
+        if (doublePoints === true) {
+          this.points += 2 * 2
+        } else if (triplePoints === true) {
+          this.points += 2 * 3
+        } else {
+          this.points += 2
+        }
+      } else if (
+        pointThree.includes(this.word[i]) ||
+        pointThree.includes(this.word[i].toLowerCase())
+      ) {
+        if (doublePoints === true) {
+          this.points += 3 * 2
+        } else if (triplePoints === true) {
+          this.points += 3 * 3
+        } else {
+          this.points += 3
+        }
+      } else if (
+        pointFour.includes(this.word[i]) ||
+        pointFour.includes(this.word[i].toLowerCase())
+      ) {
+        if (doublePoints === true) {
+          this.points += 4 * 2
+        } else if (triplePoints === true) {
+          this.points += 4 * 3
+        } else {
+          this.points += 4
+        }
+      } else if (
+        pointFive.includes(this.word[i]) ||
+        pointFive.includes(this.word[i].toLowerCase())
+      ) {
+        if (doublePoints === true) {
+          this.points += 5 * 2
+        } else if (triplePoints === true) {
+          this.points += 5 * 3
+        } else {
+          this.points += 5
+        }
+      } else if (
+        pointEight.includes(this.word[i]) ||
+        pointEight.includes(this.word[i].toLowerCase())
+      ) {
+        if (doublePoints === true) {
+          this.points += 8 * 2
+        } else if (triplePoints === true) {
+          this.points += 8 * 3
+        } else {
+          this.points += 8
+        }
+      } else if (
+        pointTen.includes(this.word[i]) ||
+        pointTen.includes(this.word[i].toLowerCase())
+      ) {
+        if (doublePoints === true) {
+          this.points += 10 * 2
+        } else if (triplePoints === true) {
+          this.points += 10 * 3
+        } else {
+          this.points += 10
+        }
```

Comment:
Similar to what Dan commented above, there is a lot of repetition here checking if doublePoints or triplePoints is true. You could refactor by replacing those two variables by a single one holding a multiplier value and then multiply the points you are adding by the current multiplier.